src/compas_fab/robots/robot.py

```diff
@@ -1634,10 +1627,11 @@ def draw(self):
         """Alias of :meth:`draw_visual`."""
         return self.draw_visual()
 
-    def draw_attached_tool(self):
```

Comment:
Is this gone for good? or moved to the robot model artists?

Reply:
There isn't currently on compas_fab master or compas master a  'draw_attached_tool` method on any of the artists, so I'm not sure what this function's up to.

Reply:
So, from appearances, not so much gone for good as much as never (?) there.

Reply:
aha! alright! then no prob!